ndarray/src/main/java/org/tensorflow/ndarray/NdArrays.java

```diff
@@ -54,8 +60,8 @@ public static ByteNdArray scalarOf(byte value) {
   /**
    * Creates a byte vector (rank 1) initialized with the given values.
    *
-   * <p>Modifying the data of the returned vector will also impact the values in the array
-   * passed in parameter.
+   * <p>Modifying the data of the returned vector will also impact the values in the array passed in
+   * parameter.
```

Comment:
Anyway we can disable auto-reformat of lines you did not modified? If not, can you tell which format schema is applied? I guess at some point, we'll need to install the spotless plugin to Java NdArray module as well.

Reply:
I am using google-java-format plugin in IntelliJ

Reply:
Ok it's not bad finally, there is not too many changes induced by it 👍